h/presenters/group_json.py

```diff
@@ -28,6 +28,23 @@ def _expand(self, model, expand):
                 model["organization"] = OrganizationJSONPresenter(
                     self.organization_context
                 ).asdict()
+        if "scopes" in expand:
+            model["scopes"] = {}
+            # The API representation of scope enforcement differs from the DB
+            # representation. All groups have an `enforce_scope` property, and
+            # it defaults to True. However, URL enforcement for incoming
+            # annotations only happens if there are 1 or more scopes to restrict
+            # to. Therefore, the API representation of this property is False
+            # if there are no scopes.
```

Comment:
This is a helpful explanation 👍

Reply:
Thanks for this feedback! I agonized a bit over these comments. I'm rather pro-comment (like, I'm OK if there are more comments than code) but worried it might be seen as overboard.